benchmarking/benchmark.py

```diff
@@ -232,11 +234,11 @@ def compose_result(status, tt, experiment, sizes, threshold):
     return result
 
 
-def delete_resources(config, credentials, run):
+def delete_resources(credentials, run):
     try:
         if run is not None and 'run_id' in run:
-            rest_client.run_delete(config['server'], credentials['project_id'], run['run_id'], credentials['result_token'])
-        rest_client.project_delete(config['server'], credentials['project_id'], credentials['result_token'])
+            rest_client.run_delete(credentials['project_id'], run['run_id'], credentials['result_token'])
+        rest_client.project_delete(credentials['project_id'], credentials['result_token'])
```

Comment:
Is it true that currently there is only server in `config`?

Reply:
No, there is lots of stuff in `config` all set around line 84. the change is because the new `rest_client` already knows the server address